frontend/src/theme/index.tsx

```diff
@@ -43,6 +43,19 @@ const customTheme = extendTheme({
         },
       },
     },
+    Modal: {
+      variants: {
+        forgotPasswordModal: {
+          dialog: {
+            maxWidth: "590px",
+            minWidth: "590px",
+            minHeight: "480px",
+            maxHeight: "480px",
+            borderRadius: "20px",
```

Comment:
I'm wondering if we should just have these defined in the modal itself rather than here - that's what we seem to be doing for the other components. Thoughts?

Reply:
Yeh that's a fair point. I really wanted to do it the Chakra UI way with defining custom styles and all but it didn't end up being as clean as I had imagined lol. I moved these inline with the component